pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -226,14 +226,19 @@ def replay_genargs_handle_dashI(arg: str, target_install_dir: str) -> str | None
         The new argument, or None to delete the argument.
     """
     assert arg.startswith("-I")
-    if (
-        str(Path(arg[2:]).resolve()).startswith(sys.prefix + "/include/python")
-        and "site-packages" not in arg
```

Comment:
You removed the `site-packages` case (where there was no replacement before this PR), I am not sure whether this was useful or not ...

Reply:
I think that part is not useful, as `site-packages` directory is in `/lib/site-packages` and not in `/include/python3.11/...`.

For header files in site-packages directories, we cover them by copying header files (e.g. numpy header files) into the virtual env directory:

https://github.com/pyodide/pyodide/blob/e492392f1b7716484a4681b707468fd191818d11/pyodide-build/pyodide_build/buildpkg.py#L534-L537

Reply:
OK makes sense, thanks!